tools/downloader/list_topologies.yml

```diff
@@ -3233,3 +3233,125 @@ topologies:
     output: "Security/object_detection/barrier/0106/dldt/"
     framework: dldt
     license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
+
+  - name: "mobilenet-v2-pytorch"
+    description: >-
+      MobileNet V2 is image classification model pretrained on ImageNet dataset. This
+      is a PyTorch implementation of MobileNetV2 architecture as described in
+      the paper "Inverted Residuals and Linear Bottlenecks: Mobile Networks for Classification,
+      Detection and Segmentation" <https://arxiv.org/pdf/1801.04381.pdf>.
+
+      The model input is a blob that consists of a single image of "1x3x224x224"
+      in RGB order. 
+
+      The model output is typical object classifier for the 1000 different classifications
+      matching with those in the ImageNet database.
+    task_type: "classification"
+    files:
+      - name: MobileNetV2.py
+        sha256: 7071e01e07e5f1b6127a2628d1be9c65cc20edfa745f4f4eb6354e7b27f5b444
+        source: https://raw.githubusercontent.com/tonylins/pytorch-mobilenet-v2/fa86a85e4026caa217fe8d62f370848195cedf4b/MobileNetV2.py
+      - name: mobilenet-v2.pth.tar
+        sha256: ecbe2b568c8602549fa9e1d5833c63848f490a48d92e5d224d1eb2063e152cf8
+        size: 14205652
+        source:
+          $type: google_drive
+          id: 1jlto6HRVD3ipNkAl1lNhDbkBp7HylaqR
+    output: "classification/mobilenet/v2/pytorch/"
+    framework: pytorch
+    pytorch_to_onnx:
+      - --model-name=mobilenet-v2
+      - --model-path=classification/mobilenet/v2/pytorch
+      - --weights=classification/mobilenet/v2/pytorch/mobilenet-v2.pth.tar
+      - --input-shape=1,3,224,224
+      - --output-file=classification/mobilenet/v2/pytorch/mobilenet-v2.onnx
```

Comment:
I don't think the output file name should be specified here, because the converter should be able to derive it automatically based on the model name and subdirectory - much like it does for Model Optimizer.

Reply:
Could you resolve this issue?

Reply:
No, because you haven't done anything about it.

Reply:
Sorry, but I don't see any sense in this change, except automation. Which advantage could we achieve with it?
Could you explain me your point?

Reply:
We know that for any model we'll need to pass an `--output-file` option to the ONNX converter. We also know that the value for said option will always be `<conv_dir>/<model_name>.onnx`. Specifying this option manually is thus of no benefit and serves only to invite mistakes. It should instead be automatically supplied by the Model Converter.

Reply:
It makes sense.
But we must go further in that case and get rid of almost all mo args, like `--framework`, `--input-model` (or any `--input_*`), `--data_type` with same reason. Would you do this?

Reply:
I was indeed planning on getting rid of `--framework` and `--data_type` (the latter I would have to remove anyway to add support for multiple output precisions). `--input_*` cannot be removed right now, because there's no way to automatically derive the values of these options.

Reply:
I cannot agree about `--input_*` with you. I randomly picked up some models and most of them have keys like `--input_*=$dl_dir/<model_name>.<format>`. Only archived models make an exception. So, I supposed that these data are derivable as well. 
In this case, we can indeed lower the tendency for mistakes in configuration. But if changes will be only in part, which you suggest, benefit would turn low. In fact, it would cause misunderstanding in "where is onnx model come from?" for MO argument `--input-model`.

Reply:
> In fact, it would cause misunderstanding in "where is onnx model come from?" for MO argument --input-model.

OK, that's a fair point. Let keep it as-is for now, then.